ConfigVersionControl/git_version_control.py

```diff
@@ -15,19 +15,18 @@
 # http://opensource.org/licenses/eclipse-1.0.php
 
 import os
```

Comment:
Import appears to be unused

Reply:
It's not, I'm using `os.path` in a few places here. Not sure why it gets highlighted, I have seen a few people asking the same question online but have not come across a satisfying answer so far